### ISSUE TITLE
Clicking on warnings should open the file and scroll to the line it originated from

### ISSUE THREAD

**Issue by Zylann** [godotengine/godot#22228]
Godot 3.1 alpha 1

I'm doing testing of a prototype with this version of Godot, and I quickly got a lot of various warnings. But I noticed that double-clicking them doesn't take me to the place they originated from, that would save time if it would.

Note: some of the warnings have a 1-line stack trace, and clicking on that stack trace leads me to the file. 
![image](https://user-images.githubusercontent.com/1311555/45711206-b57ee380-bb80-11e8-8ad4-4a7d9e3a39e2.png)

But many others don't have a stack trace, despite having a file and line number in their tooltip.
![image](https://user-images.githubusercontent.com/1311555/45711218-c16aa580-bb80-11e8-9155-e9b4d01e182a.png)

**Comment by Ranoller** [comment]
I vote for that very strong.

**Comment by dreamsComeTrue** [comment]
Is that error still exist? In current build I am seeing a lot of warnings, but ALL of them have associated GDScript line which is revelad after clicking on them. @Zylann - do you have some kind of test project that could be considered as a good example of that bug?

**Comment by tom-jk** [comment]
Testing on linux mint 18.3

Can reproduce in 3.1.alpha1
Can't reproduce in 3.1.alpha2 thru 3.2.3.rc3

<details><summary>Testing with this script (produces most of the warnings showable in the script editor in 3.1):</summary>

```gdscript
extends Node2D

const function_conflicts_constant = 5

signal unused_signal

var unused_class_variable
var function_conflicts_variable


func _ready():
	var unused_variable
	var unassigned_variable
	
	var unassigned_variable_op_assign:int
	if false:
		unassigned_variable_op_assign /= unassigned_variable
	
	unassigned_variable #standalone_expression
	
	unassigned_variable = void_assignment_function()
	
	var narrowing_conversion:int = 1 * 5.0
	
	print("fish" if randf()<0.5 else 5)
	
	return_value_discarded(32)
	
	var integer_dividend:int = 3
	var integer_divisor:int = 5
	print(integer_dividend / integer_divisor)
	
	var m = RigidBody2D.new()
	print(m.get_bounce())
	
	return
	print("unreachable")


func function_conflicts_variable(unused_argument):
	pass


func function_conflicts_constant():
	pass


func variable_conflicts_function():
	pass
var variable_conflicts_function


func void_assignment_function() -> void:
	pass


func function_used_as_property():
	pass


func return_value_discarded(arg) -> int:
	var a = 10 * arg
	return a

```

</details>

In 3.1.alpha1, the warnings in the script editor's warnings pane can be clicked to go to the correct line, but not those in the debugger errors tab (except those like the deprecated method warning).

In 3.1.alpha2, the debugger errors tab has been updated: there is no stack trace pane (they are folded under each warning/error instead). Clicking the warning/error switches to the script editor and takes you to that line.

lgtm

**Comment by akien-mga** [comment]
Thanks, closing as fixed in 3.1.
